crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -123,6 +123,27 @@ module deepbook::clob_v2 {
         price: u64
     }
 
+    /// A struct to make all orders canceled a more effifient struct
+    struct AllOrdersCanceledComponent<phantom BaseAsset, phantom QuoteAsset> has copy, store, drop {
```

Comment:
Seems like `AllOrdersCanceledComponent` has the exact same structure as `OrderCanceled`?

Reply:
the pool_id gets taken out so that it's not repeated every single time

Reply:
Naming nit: I would call this `CanceledOrder`, and call `AllOrdersCanceled` `CanceledOrders`